src/FSharpSource.Nuget.targets

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) Microsoft Open Technologies, Inc.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup Condition="'$(References_Nunit)' == 'true'">
+    <NunitFrameworkPath>$(MSBuildThisFileDirectory)..\packages\NUnit.2.6.4\lib\nunit.framework.dll</NunitFrameworkPath>
+    <NunitUtIlsPath>$(MSBuildThisFileDirectory)..\packages\NUnit.Runners.2.6.4\tools\lib\nunit.util.dll</NunitUtIlsPath>
+    <NugetRestore>true</NugetRestore>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(NugetRestore)' == 'true'">
```

Comment:
i dont think is a good idea to have this custom property.
Is pretty useless and non standard, a problem for maintenance 
The nunit nuget can be easy updated for entire solution, using standard visual studio